.github/workflows/pr-checks.yml

```diff
@@ -65,6 +65,7 @@ jobs:
                 feature:alert
                 feature:exit-form
                 feature:select-subject-age-group
+                feature:data-generator
```

Comment:
Would it make sense to put it under a different top-level package to separate dev/testing modules from the actual product features? Something like `testing:data-generator` or something like that.